src/main/java/com/iota/iri/model/persistables/Transaction.java

```diff
@@ -35,6 +45,12 @@
     //public boolean confirmed = false;
     public boolean parsed = false;
     public boolean solid = false;
+
+    /**
+     * This flag indicates if the transaction is a coordinator issued milestone.
+     */
+    public boolean isSnapshot = false;
```

Comment:
localSnapshot

since we don't change `snapshot` ...

Reply:
actually this should in fact be isMilestone - i just kept the old naming scheme (snapshotIndex) O_O